Cargo.toml

```diff
@@ -22,14 +22,14 @@ ic-certified-map = { workspace = true }
 ic-cdk = { workspace = true }
 ic-cdk-macros = { workspace = true }
 ic-eth = { workspace = true }
+cketh-common = { git = "https://github.com/rvanasa/ic", branch = "evm-rpc-canister", package = "ic-cketh-minter" } # TODO: refactor
```

Comment:
A TODO in the code should link to a Jira ticket.   :slightly_smiling_face:

Reply:
@dfx-json, do we want to use Jira (either via an epic or individual issues) or GitHub issues on this repository? My first inclination would be to use GitHub so that community members could weigh in on implementation details at any point during development.